gensim/test/test_summarization.py

```diff
@@ -148,6 +148,26 @@ def test_keywords_runs(self):
         kwds_lst = keywords(text, split=True)
         self.assertTrue(len(kwds_lst))
 
+    def test_mz_keywords(self):
```

Comment:
Travis hang because your implementation is really slow (test don't finish successfully for 10 minutes), you can make several changes
1. Try to optimize `mz_keywords`
2. Use really small corpus (micro-sample from current dataset for example)

Reply:
I'll try using the first 10240 words.